sips/sip-010/sip-010-fungible-token-standard.md

```diff
@@ -0,0 +1,132 @@
+## Preamble
+
+- Sip Number: 010
+- Title: Standard Trait Definition for Non-Fungibe Tokens
+- Author: Hank Stoever (hstove@gmail.com)
+- Consideration: Technical
+- Type: Standard
+- Status: Draft
+- Created: 2021-01-25
+- License: CC0-1.0
+- Sign-off:
+
+## Abstract
+
+Fungible tokens are digital assets that can be sent, received, combined, and divided. Most forms of currency and cryptocurrencies are fungible tokens. They have become a building block of almost all blockchains. This SIP aims to provide a flexible and easy to implement standard that can be used by developers on the Stacks blockchain when creating their own tokens.
+
+## License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at https://creativecommons.org/publicdomain/zero/1.0/
+This SIP’s copyright is held by the Stacks Open Internet Foundation.
+
+## Introduction
+
+Perhaps the oldest, and most well known, standard for fungible tokens is Ethereum's [ERC20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) standard. It has become one of the strongest building blocks for the Ethereum ecosystem. When all fungible tokens follow the same standard, any wallet or application developer can interact with it without having to create custom logic for handling each individual token.
```

Comment:
Editing -- please consider moving this paragraph to the `Related Work` section, since it discusses another similar standard from a different blockchain.  I don't think this SIP is concerned with Ethereum compatibility or copying/porting Ethereum's standards, right?